spec/components/search_spec.rb

```diff
@@ -42,4 +42,14 @@ def component_name
     render_component(size: "large")
     assert_select ".gem-c-search.gem-c-search--large"
   end
+
+  it "renders a search box with a default name" do
+    render_component(id: "my-unique-id")
```

Comment:
I was briefly confused by this, although it makes sense. Maybe (if possible) have this test pass no parameters, to make it clear?